src/QMCDrivers/SimpleFixedNodeBranch.cpp

```diff
@@ -782,6 +783,11 @@ void SimpleFixedNodeBranch::read(const std::string& fname)
     }
   }
 
+  if (BranchMode[B_DMC] && !bmode[B_DMC])
```

Comment:
When troubleshooting https://github.com/QMCPACK/qmcpack/issues/4487, I noticed that the added blocker blocks DMC-> any QMC  because bmode[B_DMC] is always 0.